backend/btrixcloud/models.py

```diff
@@ -1423,6 +1423,14 @@ class PreloadResource(BaseModel):
     crawlId: str
 
 
+# ============================================================================
+class OriginCount(BaseModel):
+    """Origin Count"""
+
+    origin: str
+    count: int
+
+
 # ============================================================================
 class Collection(BaseMongoModel):
```

Comment:
Should we include `topPageOrigins` in the base Collection model too, since we're storing it there? Might be useful for e.g. org import/export to have it on the base model.

Reply:
Yeah not a bad idea to rename internally, and I would keep the prefix I think